### PR TITLE
W-13630523 1.5 coverage status gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc [hunk 1]
```diff
@@ -70,6 +70,13 @@ Users with *View Policies* role for the environment will able to see the list of
 // +
 // If an application is deployed to an environment which has an automated policy that doesn't cover the deployed runtime, for example with an automated policy affecting a runtime range from 4.1.2 to 4.1.4, and a user deploys to a 4.1.5 runtime, a notification will be sent to the organization, and environment administrators, to let them know of this activity.
 
+=== Viewing Coverage Status
+
+To view what API instances are covered by an automated policy:
+
+. Navigate to *Anypoint Platform* > *API Manager*.
+. In API Administration, click *Automated Policies*.
+. Click more options button (image:more-options-menu.png[1%,1%]) of the policies whose coverage you want to view.
```
Comment (chris-campbell9) [inline gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc]:
```suggestion
. Click the more options button (image:more-options-menu.png[1%,1%]) for the policy whose coverage you want to view.
```
I'm thinking they should be consistent in terms of being singular.

File: gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc [hunk 2]
```diff
@@ -70,6 +70,19 @@ Users with *View Policies* role for the environment will able to see the list of
 // +
 // If an application is deployed to an environment which has an automated policy that doesn't cover the deployed runtime, for example with an automated policy affecting a runtime range from 4.1.2 to 4.1.4, and a user deploys to a 4.1.5 runtime, a notification will be sent to the organization, and environment administrators, to let them know of this activity.
 
+=== Viewing Coverage Status
+
+View the *Coverage status* of an API instances to find if an automated policy covers your API instance or why it doesn't. Typically, instances are not covered due to their runtime type or runtime version.
```
Comment (kevintroller) [inline gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc]:
```suggestion
View the *Coverage status* of an API instance to find out if an automated policy covers your API instance or why it doesn't. Typically, instances are not covered due to their runtime type or runtime version.
```

File: gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc [hunk 3]
```diff
@@ -70,6 +70,19 @@ Users with *View Policies* role for the environment will able to see the list of
...
+
+To view what API instances are covered by an automated policy:
+
+. Navigate to *Anypoint Platform* > *API Manager*.
+. In API Administration, click *Automated Policies*.
```
Comment (kevintroller) [inline gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc:80]:
```suggestion
. In *API Administration*, click *Automated Policies*.
```

**Review by kevintroller — DISMISSED** [review]
Added minor comments

**Review by kevintroller — DISMISSED** [review]

**Review by kevintroller — APPROVED** [review]

**Pull request merged**